scriptmodules/supplementary/skyscraper.sh

```diff
@@ -44,14 +45,19 @@ function install_skyscraper() {
         'artwork.xml.example2'
         'artwork.xml.example3'
         'artwork.xml.example4'
+        'platforms.json'
+        'screenscraper.json'
         'tgdb_developers.json'
         'tgdb_publishers.json'
         'mameMap.csv'
         'aliasMap.csv'
-        'hints.txt'
+        'hints.xml'
+        'docs'
         'import'
         'resources'
         'cache/priorities.xml.example'
+        'VERSION'
+        'skyscraper.pro'
```

Comment:
This file is not needed during runtime.

Reply:
It's used during `configure` to get the list of files, which means it needs to exist in `$md_inst`.

Or is that the whole point of the fallback list: we read from the file in `$md_build` (if building from source) or, if that file doesn't exist (because binary install), use the fallback list instead?